### PR TITLE
add nanorc to packaged builds

### PR DESCRIPTION
**Pull request by jessfraz**
ping @tianon

### CODE REVIEW

**Comment by jfrazelle** [comment]
oinng @tianon :angel:

File: hack/make/.build-deb/docker-engine.install [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #contrib/syntax/vim/doc/* /usr/share/vim/vimfiles/doc/
 #contrib/syntax/vim/ftdetect/* /usr/share/vim/vimfiles/ftdetect/
 #contrib/syntax/vim/syntax/* /usr/share/vim/vimfiles/syntax/
+contrib/syntax/nano/Dockerfile.nanorc /usr/share/nano/Dockerfile.nanorc
```
Comment (tianon) [inline hack/make/.build-deb/docker-engine.install]: Can you `:sort` this file? :angel:

Comment (tianon) [inline hack/make/.build-deb/docker-engine.install]: Also, drop the `Dockerfile.nanorc` on the RHS here:
```
contrib/syntax/nano/Dockerfile.nanorc /usr/share/nano/
```

Comment (tianon) [inline hack/make/.build-deb/docker-engine.install]: Oh gee, and those comments are wrong, this shouldn't have a leading `/` IIRC.

Comment (jfrazelle) [inline hack/make/.build-deb/docker-engine.install]: will do

Comment (tianon) [inline hack/make/.build-deb/docker-engine.install]: :heart:

Comment (jfrazelle) [inline hack/make/.build-deb/docker-engine.install]: updated!

**Comment by tianon** [comment]
LGTM :heart:

Should we wait for `janky`, or just merge since it isn't even running or checking these scripts?

**Pull request merged**